Adds Delete button and action to collections#show for HYDRASIR-16

scherztc opened the pull request.

jeremyf commented:
Please squash the commits together, as they represent one unit.

Also amend your commit message to better conform to https://github.com/projecthydra/curate/blob/develop/CONTRIBUTING.md#well-written-commit-messages

I don't have this in the CONTRIBUTING.md, but I would like to see HYDRASIR-16 on a new line

scherztc committed:
Adds delete button to the collections#show view

Although the collections#index view had a delete button, the
collections#show view was missing the delete button.  Copied
the button_to code from #index to #show.  Modified the
spec/features/collections_show_spec.rb to have a visit test.

HYDRASIR-16 #close

scherztc commented:
Cleaned up submitted code to adhere to standards  https://github.com/projecthydra/curate/blob/develop/CONTRIBUTING.md#ruby-file-structure
- resolved git blame potential
- set tabs to 2
- adds trailing line to file
- Proper commit message
- Created passable spec test, but could still use some help here.
